### PR TITLE
Onboarding web/CertificateRegistration for autogeneration

### PR DESCRIPTION
**Pull request by hunmix**
```
> azure-schema-generator@1.0.0 generate-single D:\code\azure-resource-manager-schemas\generator
> ts-node cmd/generatesingle "web/resource-manager"

[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git fsck
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git reset -q .
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git checkout -q -- .
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git clean -q -fd
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git gc
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git fetch -q
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git checkout -q origin/master
Using whitelist config:
{
  "basePath": "web/resource-manager",
  "namespace": "Microsoft.CertificateRegistration"
}
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\917l2pn6gea --tag=all-api-versions --api-version=2015-08-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\web\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
WARNING: Found duplicate definition for variable-named type Microsoft.CertificateRegistration/certificateOrders. Skipping definition with path '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CertificateRegistration/certificateOrders/{name}'.
WARNING: Found duplicate definition for variable-named type Microsoft.CertificateRegistration/certificateOrders. Using definition with path '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CertificateRegistration/certificateOrders/{certificateOrderName}'.
WARNING: Found duplicate definition for variable-named type Microsoft.CertificateRegistration/certificateOrders/certificates. Skipping definition with path '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CertificateRegistration/certificateOrders/{certificateOrderName}/certificates/{name}'.
WARNING: Found duplicate definition for variable-named type Microsoft.CertificateRegistration/certificateOrders/certificates. Using definition with path '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CertificateRegistration/certificateOrders/{certificateOrderName}/certificates/{name}'.
[553.46 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2015-08-01\Microsoft.CertificateRegistration.json
Provider Namespace: Microsoft.CertificateRegistration
API Version: 2015-08-01
Resource Types (Resource Group Scope):
- certificateOrders
- certificateOrders/certificates
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\yfwsc6n5c3a --tag=all-api-versions --api-version=2018-02-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\web\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
WARNING: Skipping path '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/host/default/{keyType}/{keyName}': Parameter reference {keyType} is not defined as an enum
i-versions --api-version=2019-08-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\web\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
```

### CODE REVIEW

File: schemas/common/autogeneratedResources.json [hunk 1]
```diff
@@ -2831,25 +2858,28 @@
           "$ref": "https://schema.management.azure.com/schemas/2020-01-01/Microsoft.MachineLearningServices.json#/resourceDefinitions/workspaces_privateEndpointConnections"
         },
         {
-          "$ref": "https://schema.management.azure.com/schemas/2017-05-01-preview/Microsoft.MachineLearningExperimentation.json#/resourceDefinitions/accounts"
+          "$ref": "https://schema.management.azure.com/schemas/2018-06-01-preview/Microsoft.Maintenance.json#/resourceDefinitions/maintenanceConfigurations"
```
Comment (leni-msft) [inline schemas/common/autogeneratedResources.json]: These changes looks unrelated? you might need reset your working branch first

**Comment by hunmix** [comment]
regenerate in https://github.com/Azure/azure-resource-manager-schemas/pull/968
